Sprint-2/errors/0.js

```diff
@@ -4,6 +4,12 @@
 // interpret the error message and figure out why an error is occurring
 
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+  str = `${str[0].toUpperCase()}${str.slice(1)}`;
   return str;
 }
+
+console.log(capitalise("hello"));
+
+// it gives an error "str has already been declared" so we don't need to define it again in line 7
+// in line 6 - we give the function the value or variable so in line 7- we don't nee to define the variable 
```

Comment:
Yes correct, `str` in line 6 is the function's 'parameter'

Reply:
thats right